PostHog/posthog
# fix: floating uninstall button

## Problem


https://github.com/PostHog/posthog/issues/18029

## Changes
1. Remove default relative position for Lemon buttons




## How did you test this code?
In the browser

<img width="392" alt="Screenshot 2023-10-17 at 14 46 49" src="https://github.com/PostHog/posthog/assets/26106822/f1cbc6e7-d84e-4c05-aed7-2282917b580f">


1. Navigate to `Browse Apps` > `Apps Management`
1. Shrink the screen to mobile size
1. Click on the humburger icon to reveal the sidebar
1. Ensure the Uninstall button isn't floating around.
